common/pom.xml

```diff
@@ -25,5 +29,24 @@
             <artifactId>gson</artifactId>
             <version>${gson.version}</version>
         </dependency>
+
+        <!-- Jackson dependency for parsing yaml files-->
```

Comment:
Jackson is a bit on the heavy side for what it's being used, is it possible to switch to SnakeYAML which is much lighter?

Reply:
sure, ill try that